src/sentry/static/sentry/app/data/forms/accountAppearance.jsx

```diff
@@ -0,0 +1,64 @@
+import {createSearchMap} from './util';
+import timezones from '../timezones';
+import languages from '../languages';
+
+const forms = [
+  {
+    // Form "section"/"panel"
+    title: 'Events',
+    fields: [
+      {
+        name: 'stacktrace_order',
+        type: 'choice',
+        required: false,
+        choices: [
+          ['-1', 'Default (let Sentry decide)'],
```

Comment:
i wish we had a better way to share constants between python and js at build time

Reply:
yeah... it should be easily doable, e.g. if constants were in json